hw_12/task_1.js

```diff
@@ -0,0 +1,101 @@
+// 1. Создайте функцию delayTwoSeconds, принимающую на вход коллбэк функцию, 
+//которая будет отрабатывать спустя 2 секунды после вызова delayTwoSeconds
+function delayTwoSeconds(callback) {
+    setTimeout(() => {
+        callback();
+    }, 2000);
+}
+delayTwoSeconds(() => {console.log('wait 2 seconds for function');
+});
```

Comment:
Не всегда нужньі скобки
```suggestion
function delayTwoSeconds(callback) {
    setTimeout(() => callback(), 2000); // or setTimeout(callback, 2000);
}
delayTwoSeconds(() => console.log('wait 2 seconds for function'));
```